content/docs/2.9/concepts/scaling-deployments.md

```diff
@@ -210,6 +210,8 @@ Starting from Kubernetes v1.18 the autoscaling API allows scaling behavior to be
 Trigger fields:
 - **type**: The type of trigger to use. (Mandatory)
 - **metadata**: The configuration parameters that the trigger requires. (Mandatory)
+- **name**: An optional name for this trigger. This value can be used to easily distinguish this specific trigger and it's metrics when consuming [Prometheus metrics](../operate/prometheus.md). (Optional)
```

Comment:
Optional is already refered to at the end.
```suggestion
- **name**: Name for this trigger. This value can be used to easily distinguish this specific trigger and its metrics when consuming [Prometheus metrics](../operate/prometheus.md). (Optional)
```

Also, what happens if it's not specified; what value will it use?